File: data/activity1/scripts/getResults.py
#!/usr/bin/env python3
import os
import sys
import glob

data = {}

fileList = glob.glob("../data/file*.txt")
for i in range(0,len(fileList)):
    InFile = open (fileList[i], 'r')
    for Line in InFile:
        Line = Line.strip('\n')
        ElementList = Line.split('\t')
        if (len(ElementList) > 1) and (ElementList[0] != "Parameter"):
            data[(i,ElementList[0])] = ElementList[1]

print('File\tParameter\tValue',end='\n')
for j,k in sorted(data.keys()):
    print(fileList[j],'\t',k,'\t',data[(j,k)],end='\n')
exit;

File: introduction/scripts/getResults.pl
#!/usr/bin/perl -w

%data = ();

@fileList = glob("../data/*.params");
for $i (0..(scalar(@fileList)-1))
{
#    print "$fileList[$i]\n";
    open FH1,'<',"$fileList[$i]";
    while(<FH1>)
    {
	if (/(\S+)\s+(\S+)/)
	{
	    $parameter = $1;
	    $value = $2;
	    if ($parameter ne "Parameter")
	    {
		push @{$data{$parameter}}, $value;
	    }
	}
    }
    close FH1;
}

print "File";
foreach $parameter (sort(keys(%data)))
{
    print "\t$parameter";
}
print "\n";

for $i (0..(scalar(@fileList)-1))
{
    print "$fileList[$i]";
    foreach $parameter (sort(keys(%data)))
    {
	print "\t$data{$parameter}[$i]";
    }
    print "\n";
}
exit;
